Makefile.toml

```diff
@@ -259,3 +259,16 @@ description = "Serve the schema.golem.cloud directory locally to help testing th
 install_crate = "miniserve"
 command = "miniserve"
 args = ["--interfaces", "127.0.0.1", "--port", "41357", "schema.golem.cloud"]
+
+## Agent template management
+[tasks.rebuild-ts-agent-template]
```

Comment:
This is a manual step on purpose. We will run it whenever we want to update the "base wasm" for agents, and the compiled WASM itself is part of the template. There is no need to do this on every build.